### PR TITLE
Update to the SRP GA library

### PR DESCRIPTION
**Pull request by yaxia**
This is the change to update to the SRP GA library and support the GA features.
* Support custom domain in: azure account set/show
* Support account tags in azure account set/show
* Support checking name availability: azure account check

PS: I am not sure why the change shows so many commits. May be due to I merged from a branch whose name duplicates from the branch that created and deleted months ago. 

The commits should be:
**Commits on Oct 09, 2015**
Update to use SRP GA library  65262d9
Merge branch 'dev' of https://github.com/Azure/azure-xplat-cli into SRP  cacf95a
Set the default concurrency to 10  bd8d3bd

**Commits on Oct 10, 2015**
Add --description back in the storage account set command  91096d8  

**Commits on Oct 16, 2015**
Merge from the public repo  9688b1e  

**Commits on Oct 19, 2015**
Restore the upload/download concurrency to 5  ce2918e  
Update the test recordings for SRP GA  9e95fdc  

**Commits on Oct 20, 2015**
Update VM docker test recordings  656a810
Update test recordings for the VM commands  dba36d9

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@yaxia__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (yaxia). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by yugangw-msft** [comment]
@yaxia , this time might be too late to consolidate all those commits , but next time please do routine rebase so to get rid of not useful"merge" commits
```bash
$ git fetch upstream
$ git rebase upstream/dev
``` 
You can also use "git rebase -i" to consolidate commits so to easy for people to trace code history

**Comment by yaxia** [comment]
@yugangw-msft Thanks. I've updated the PR. It looks correct now.

File: package.json [hunk 1]
```diff
@@ -55,6 +55,7 @@
     "azure-monitoring": "0.10.2",
     "azure-asm-network": "0.10.2",
     "azure-arm-resource": "0.10.6",
+    "azure-arm-storage": "~0.11.0",
```
Comment (yugangw-msft) [inline package.json]: I suggest you change to a fixed version, unless you do mean to slide to the most recent minor version. The node sdk lib was published by someone else. bot you, so it is always possible the new version might break you cmdlets

**Comment by yugangw-msft** [comment]
the rest looks good to me

**Comment by yaxia** [comment]
Resolved the comment, thanks.

**Pull request merged**